files/fr/glossary/spa/index.html

```diff
@@ -0,0 +1,38 @@
+---
+title: Application monopage (SPA)
+slug: Glossary/SPA
+translation_of: 'Glossary/SPA'
+---
+<p>Une application monopage ou SPA (« <i>Single-page application</i> » en Anglais) est une implémentation d'application Web qui ne charge qu'un seul document Web, puis met à jour le contenu du corps de ce document via des API JavaScript telles que {{domxref("XMLHttpRequest")}} et <a href="/fr/docs/Web/API/Fetch_API">Fetch</a> lorsqu'un contenu différent doit être affiché.</p>
+
+<p>Cela permet donc aux utilisateurs d'utiliser des sites Web sans charger de nouvelles pages entières à partir du serveur, ce qui peut entraîner des gains de performances et une expérience plus dynamique, avec certains inconvénients tels que le référencement (SEO), plus d'efforts nécessaires pour maintenir l'état, mettre en œuvre la navigation et effectuer des performances surveillances significatives de surveillance.</p>
+
+<h2 id="learn_more">En apprendre plus</h2>
+
+<h3 id="general_knowledge">Connaissance générale</h3>
+
+<p>Certains des frameworks SPA les plus populaires incluent:</p>
+
+<ul>
+ <li><a href="https://reactjs.org/">React</a></li>
+ <li><a href="https://angular.io/">Angular</a></li>
+ <li><a href="https://vuejs.org/">Vue.JS</a></li>
+</ul>
+
+<section id="quick_links">
```

Comment:
This ID should not be lowercased for the content to appear in the sidebar.

Reply:
ho sorry about that, thanks for the review

Reply:
no need for apologies this is quite a specific thing :)